src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -101,6 +101,14 @@ PeerImp::~PeerImp ()
     }
 }
 
+// Helper function to check for valid uint256 values in protobuf buffers
+static
+bool
+stringIsUint256Sized (std::string const& pBuffStr)
+{
+    return pBuffStr.size() == sizeof (uint256);
```

Comment:
Tiniest nitpick:  Consider `return pBuffStr.size() == uint256::size();`

This is guaranteed to be the same functionality, but seems like a little bit higher level abstraction to me.  But nothing I'll loose sleep over.